spec/requests/medications_spec.rb

```diff
@@ -153,51 +138,39 @@ def post_create(medication_params)
     let!(:valid_medication_params) { { name: 'Updated Medication Name' } }
 
     def put_update(medication_params)
-      put :update, params: { id: medication.id, medication: medication_params }
+      put medication_path(medication), params: { medication: medication_params }
     end
 
-    context 'when the user is logged in' do
-      include_context :logged_in_user
+    context 'when signed in' do
+      before { sign_in user }
 
       context 'when valid params are supplied' do
+
         it 'updates a medication' do
           put_update valid_medication_params
-          expect(assigns(:medication)[:name]).to eq(valid_medication_params[:name])
+          expect(response.body).to include(valid_medication_params[:name])
         end
 
-        context 'when refill is updated to a new date' do
-          it 'has no validation errors and has correct refill' do
-            put_update valid_medication_params.merge(refill: '12/12/2020')
-            expect(assigns(:medication).errors).to be_empty
-            expect(assigns(:medication)[:refill].strftime('%d/%m/%Y')).to eq('12/12/2020')
-          end
+        it 'redirects to medication' do
+          put_update valid_medication_params
+          expect(response).to have_http_status(302)
         end
 
         context 'when refill is update to an empty value' do
-          it 'has no validation errors and uses auto-generated refill' do
+          it 'redirects to medication' do
             put_update valid_medication_params.merge(refill: nil)
-            expect(assigns(:medication).errors).to be_empty
-            expect(assigns(:medication)[:refill]).to be_between(assigns(:medication)[:updated_at].to_date,
-              ((7 * assigns(:medication)[:total]) / (assigns(:medication)[:dosage] * assigns(:medication)[:weekly_dosage].count)).days.from_now)
+            expect(response).to have_http_status(302)
           end
         end
-
-        it 'redirects to the medication page' do
-          put_update valid_medication_params
-          expect(response).to redirect_to medication_path(assigns(:medication))
-        end
       end
 
       context 'when invalid params are supplied' do
-        let(:invalid_medication_params) { valid_medication_params.merge(name: nil, dosage: nil) }
-        before { put_update invalid_medication_params }
-
-        it 're-renders the new medication form' do
-          expect(response).to render_template(:new)
-        end
-
-        it 'adds errors to the medication ivar' do
-          expect(assigns(:medication).errors).not_to be_empty
+        let(:invalid_medication_params) { 
+          valid_medication_params.merge(name: nil, dosage: nil)
+        }
```

Comment:
https://github.com/rubocop-hq/ruby-style-guide#single-line-blocks-delimiters
```suggestion
        let(:invalid_medication_params) do
          valid_medication_params.merge(name: nil, dosage: nil)
        end
```